gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -207,6 +209,160 @@ def __repr__(self):
         [str(s) for s in [self.name, self.genotype, self.phaseset, self.info]])
 
 
+class ToVcfRecordCoder(coders.Coder):
+  """Coder for encoding :class:`Variant` objects as .vcf text lines."""
+
+  def _get_variant_info(self, variant):
+    info_list = []
+    # Set END in info if it doesn't match start+len(reference_bases). This is
+    # usually the case for non-variant regions.
+    if (variant.start and variant.reference_bases and
+        variant.start + len(variant.reference_bases) < variant.end):
```

Comment:
Sounds good! Do we also want to add an `and variant.end`. Since `variant.end` isn't necessary to print vcf lines so this would allow the function to not break when it is None.